src/view/frontend/web/css/tailwind/components/modal.css

```diff
@@ -0,0 +1,10 @@
+.modal {
+    /*
+     * TODO: add tailwind classes used for the cart and modal styles
+     * Makeing the modal and offcanvs styles theme specific and mor adjustable
```

Comment:
Makeing the modal and offcanvs styles theme specific and mor adjustable > Making the modal and off-canvas styles theme specific and more adjustable

Reply:
@wigman I copy pasted your css file from the tailwind section. so this mistake is there in Hyvä Themes too?

Reply:
Lol. you're right. So we can remove it entirely from here, and I'll revisit that in the main theme :)

Reply:
Corrected it